prow/config/config.yaml

```diff
@@ -1498,6 +1498,22 @@ branch-protection:
                   - 'license/cla'
                   - 'idc-jenkins-ci-tispark/integration-test'
                 strict: true
+      tiunimanager:
+        branches:
+          main:
+            protect: true
+            required_status_checks:
+              contexts:
```

Comment:
Maybe `jenkins-tiem/verify` need to be required?

cc: @haiboumich, please confirm it.

Reply:
> Maybe `jenkins-tiem/verify` need to be required?
> 
> cc: @haiboumich, please confirm it.

confirm